sat/cli/status/status_module.py

```diff
@@ -370,6 +370,8 @@ def map_heading(heading):
 
     @staticmethod
     def include_heading(heading, *, component_type, **_):
+        if component_type != 'Node' and heading == 'Arch':
+            return False
         return component_type == 'Node' or heading not in ['NID', 'Role', 'SubRole']
```

Comment:
This looks like it will work, but I believe it could be simplified. The intent of this code before this change is to include all headings for the "Node" type and to exclude certain headings for all other types. The heading "Arch" could just be added to the list like this:

```suggestion
        return component_type == 'Node' or heading not in ['NID', 'Role', 'SubRole', 'Arch']

```